packages/x-date-pickers/src/internals/hooks/usePicker/usePickerValue.types.ts

```diff
@@ -344,7 +340,31 @@ export interface UsePickerValueContextValue {
    */
   setOpen: React.Dispatch<React.SetStateAction<boolean>>;
   /**
-   * `true` if the picker is open, `false` otherwise.
+   * Set the current value of the picker to be empty.
+   * The value will be `null` on single pickers and `[null, null]` on range pickers.
    */
-  open: boolean;
+  clearValue: () => void;
+  /**
+   * Set the current value of the picker to be the current date.
+   * The value will be `today` on single pickers and `[today, today]` on range pickers.
+   * With `today` being the current date, with its time set to `00:00:00` on Date Pickers and its time set to the current time on Time and Date Pickers.
+   */
+  setValueToToday: () => void;
+  /**
+   * Accept the current value of the picker.
+   * Will call `onAccept` if defined, and if the picker is closed, this value will be retained.
```

Comment:
```suggestion
   * Will call `onAccept` if defined. If the picker is closed, this value will be retained.
```

Reply:
```suggestion
   * Will call `onAccept` if defined.
   * If the picker is re-opened, this value will be the one used for initialize the views.
```

My sentence made no sense at all :laughing: